draft-ietf-tls-tls13.md

```diff
@@ -610,8 +610,8 @@ draft-20
 
 - Define how RFC 7250 works (*).
 
-- Re-enable post-handshake client authentication even when you do PSK
-  (this was just editor error).
+- Re-enable post-handshake client authentication even when you do PSK.
+  Previous prohibition was editor error.
```

Comment:
The pedant in me is noting that "previous prohibition" needs an article in order to make a complete sentence, but I am not convinced that a complete sentence is strictly needed in this context.